java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture2/tasks/_1/Number1.java

```diff
@@ -0,0 +1,266 @@
+        }
+    }
+
+    void calculations() {
+
+        Calculator calculator = new Calculator();
+
+        RangeOfNumber rangeOfNumber = new RangeOfNumber();
+
+        int numberOne = 0, numberTwo;
+        double numberOneDouble, numberTwoDouble;
+        char operation;
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Input first number:");
+
+        //MARK: IF INTEGER
+        if (scanner.hasNextInt()) {
+            calculator.isInteger = true;
+            numberOne = scanner.nextInt();
+            calculator.setCurrentValueInt(numberOne);
+
+            System.out.println("Choose operation:");
+            operation = scanner.next().charAt(0);
+            System.out.println("1. Input second integer:\n2. Get current value");
+
+            if (scanner.nextInt() == 2) {
+                calculator.getCurrentValue();
+            }
+            System.out.println("Input second integer");
+            numberTwo = scanner.nextInt();
+            calculator.setNumberOne(numberOne);
+            calculator.setNumberTwo(numberTwo);
+
+            rangeOfNumber.rangeOfNumberInt(numberOne, numberTwo);
+
+            if (!rangeOfNumber.outOfRange) {
+                switch (operation) {
+                    case '+' -> calculator.addition();
+                    case '-' -> calculator.subtraction();
+                    case '*' -> calculator.multiplication();
+                    case '/' -> calculator.division();
+                    default -> System.out.println("Error");
+                }
+            }
+        }
+
+        //MARK: IF DOUBLE
+        else if (scanner.hasNextDouble()) {
+            calculator.isInteger = false;
+            numberOneDouble = scanner.nextDouble();
+            calculator.setCurrentValueDouble(numberOneDouble);
+
+            System.out.println("Choose operation:");
+            operation = scanner.next().charAt(0);
+            System.out.println("1. Input second double:\n2. Get current value");
+
+            if (scanner.nextInt() == 2) {
+                calculator.getCurrentValue();
+            }
+            System.out.println("Input second double");
+
+            numberTwoDouble = scanner.nextDouble();
+
+            calculator.setNumberOneDouble(numberOneDouble);
+            calculator.setNumberTwoDouble(numberTwoDouble);
+            rangeOfNumber.rangeOfNumberDouble(numberOneDouble, numberTwoDouble);
+
+            if (!rangeOfNumber.outOfRange) {
+                switch (operation) {
+                    case '+' -> calculator.addition();
+                    case '-' -> calculator.subtraction();
+                    case '*' -> calculator.multiplication();
+                    case '/' -> calculator.division();
+                    default -> System.out.println("ERROR");
+                }
+            }
+        }
+    }
+}
+
+class Calculator {
+
+    Number1 number1 = new Number1();
+
+    boolean isInteger;
+    private double calculationForDouble;
```

Comment:
Why do we need to have calculationForDouble/calculationForInteger and currentValueInt/currentValueDouble variables? as for me, they duplicate each other